backend/BalanceBuddy/src/main/java/com/example/balancebuddy/config/KeyUtils.java

```diff
@@ -27,79 +27,103 @@ public class KeyUtils {
     @Autowired
     Environment environment;
 
+    // Path to the private key for access tokens
     @Value("${ACCESS_TOKEN_PRIVATE_KEY_PATH}")
     private String accessTokenPrivateKeyPath;
 
+    // Path to the public key for access tokens
     @Value("${ACCESS_TOKEN_PUBLIC_KEY_PATH}")
     private String accessTokenPublicKeyPath;
 
+    // Path to the private key for refresh tokens
     @Value("${REFRESH_TOKEN_PRIVATE_KEY_PATH}")
     private String refreshTokenPrivateKeyPath;
 
+    // Path to the public key for refresh tokens
     @Value("${REFRESH_TOKEN_PUBLIC_KEY_PATH}")
     private String refreshTokenPublicKeyPath;

```

Comment:
In this case is clear that these variables are representing some paths for the keys. You can delete the comments. The role of the comments is to explain pieces or sequences of code that don't seem very clear or intuitive at first glance, such as a more complex business logic function.